### ISSUE TITLE
anomaly in uploading pdf in media

### ISSUE THREAD

**Issue by sryu** [yuzoolcode/pulsecms-tracking#48]
I am experiencing inconsistencies in pushing pdf files into media. 

I am using pulse v51b3 and b4. pulse is installed as a subdirectory in the root dir, and functions as expected. 

To create a depository of pdf files (along with InStack's P5 Delivery), I created a pdf folder within Media, and upload pdfs into it. 
When I upload a 4 MB pdf file, it seems to work although after uploading, the button/icon ends up showing an "x" mark, which upon hovering over says "Server responded with **500 code**". Despite that, the file is added into the pdf folder. (BTW, the original pdf had spaces in its title, and pulse seems to replace them with "_", which is OK, but not desirable ultimately, as it prevents the long title from breaking naturally when displaying the list of available downloads in a small area in a table or list, for example.  Given that when I manually changed the title back to the original form using ftp program, pulse seems to work with them just as well, so this replacement of spaces with "_" business seems just inconvenient. But that's a side issue.)

(**Another side issue is that pulse creates an empty **gallery.txt** file in the folder, which then shows up as a downloadable item among other pdfs. Not right. Also when I then try to download it, it gives out error message saying I cannot access the item, revealing detailed path to that file in the system, which is not good...)

Back to the main issue: and this is the main problem - 

When I upload a **larger file**,  (13 MB for example), pulse seems to process it, and the upload icon turns to the "check mark", and I do not see the error message like above. All seems good, but the file is nowhere to be seen. 

(.htaccess has upload and processing limits set as default, 20MB and 25MB. I had tried to increase it to 40MB and 50 MB respectively, but it didn't solve the problem.)

**Comment by ghost** [comment]
Where are you hosting the site?
You might need to increase via a server setting in the php.ini

e.g.
https://help.dreamhost.com/hc/en-us/articles/214200558-How-do-I-increase-the-PHP-upload-limit-

Is it related to this?
https://forum.pulsecms.com/t/big-file-upload-unsuccessful/982/2

Is this for PDFs only?

**Comment by sryu** [comment]
Indeed it seems the problem is related to some of the php variable settings. wading through the information overload I got from running the phpinfo script on the rootlevel of the site in question,
( www.yumibae.com/info.php ), I find the following: 

memory_limit = 256M;
upload_max_filesize = 64M;
post_max_size = 64M;
max_execution_time = 30;   // way too small here? 
max_input_vars = 1000;  // and this too? 

BTW, I think the same happened with large jpg or png files too. 

So, I think I will try to increase these using the php.ini on the hosting site (bluehost.com). If this fails, I will get back to this again.

**Comment by sryu** [comment]
I increased the variable 
max_input_time from 30 to 500. 
All others are same as above, which I believe are ample. 
Yet the uploading fails for a jpg of 7.5 MB, and a pdf of 13MB in size....
(the same jpg scaled down to 5.2 MB uploaded fine.)

BTW, in the Settings, the cryptic Media section has the following values if they are releavant:
JPEG quality = 85, resampling off = OFF, JPEG quality = 1200, Thunmbnail height = 120, 
Create thumbnails = ON.
**(can someone also explain what these are for? I understand that pulse seems to rescale images to web-friendly size after uploading, but the meaning of these numbers....?)**

(I am using the default PHP 5.6 option on bluehost, which apparently allows each directory to have its own php.ini.  Just to make sure, I updated and verified the php-varaiables are the same in both the public_html dir and the root directory of the site in question.)
www.mathgamehouse.com/info.php will get phpinfo from the public_html 
www.yumibae.com/info.php will get the same from the root dir of the site in this shared hosting setup.
